.github/workflows/release-cycle.yml

```diff
@@ -14,14 +14,22 @@ jobs:
         os: [windows-latest, ubuntu-latest]
 
     steps:
+      - name: Silence Windows warnings
+        run: git config --global core.autocrlf input
+
       - uses: actions/checkout@v3
 
       - uses: actions/setup-python@v4
         with:
           python-version: "3"
+          cache: pip
+          cache-dependency-path: .github/workflows/release-cycle.yml
+
+      - name: Install Jinja
+        run: python -m pip install jinja2
 
       - name: Generate release cycle output
-        run: python -I -bb -X dev -X warn_default_encoding -W error _tools/generate_release_cycle.py
+        run: python -I -bb -X dev -X warn_default_encoding -W error _tools/generate_release_cycle.py --today 2023-01-21
 
       - name: Check for differences
         run: |
```

Comment:
Since we're now running this script on every CI and local (`make`) build anyway to generate these files, wouldn't it make sense to removed the checked-in generated CSVs and add them to gitignore too?

And since that already checks if the script ran successfully, and the files exist, are up to date, and are valid (at least, to the extent they can be injected into and loaded by Sphinx) the generate files are always up to date (in the production site, at least), this workflow is entirely redundant (since `git diff` won't show anything on `gitignored` files) and can be removed?

Reply:
Yeah, I guess we could do that too.

@ezio-melotti Any preference on whether the CSVs are kept in the repo? 

If we later have some external need for them, we could export them to something like `_static`

(We can also deal with this in a follow-up)

---

@CAM-Gerlach About the SVG, do you think we still need the `--today` override?

Reply:
> (We can also deal with this in a follow-up)

Not sure if you're referring to this whole thing or just specifically the part about exposing the CSVs externally, but IMO the latter makes more sense to me, since the former will simplify things substantially and avoid the caveats/regressions this is introducing.

> @CAM-Gerlach About the SVG, do you think we still need the --today override?

With this workflow being removed, it won't be actively required/used in CI currently, but either we or external parties could have a potential usecase for for reproducible/user-controlled output, so I don't think it hurts to leave it in now that you've added it.

Reply:
Updated to also remove the CSVs from Git and generate them as part of the deploy.

Reply:
Gotcha. Can we also get rid of this workflow, too? Since as mentioned, it doesn't really do anything now that the script is being run as part of the normal CIs, and the files are `gitignored` so the `git diff` check will never see them anyway?

Reply:
> Updated to also remove the CSVs from Git and generate them as part of the deploy.

So now we use the json as primary source to generate both the SVG and the CSVs, right?

Reply:
![image](https://user-images.githubusercontent.com/17051931/215655375-0711bccc-edfe-4b77-9856-00447ff5ac40.png)